tests/parm/ufs.configure.s2s.IN

```diff
@@ -93,6 +97,20 @@ MED_attributes::
       coupling_mode = @[CPLMODE]
       pio_rearranger = @[pio_rearranger]
       ocean_albedo_limit = @[ocean_albedo_limit]
+      histaux_ice2med_file1_auxname = ice.1h.aux
+      histaux_ice2med_file1_doavg = .false.
+      histaux_ice2med_file1_enabled = .true.
+      histaux_ice2med_file1_flds = all
+      histaux_ice2med_file1_history_n = @[MED_history_n]
+      histaux_ice2med_file1_history_option = nhours
+      histaux_ice2med_file1_ntperfile = 9999
+      histaux_ocn2med_file1_auxname = ocn.1h.aux
+      histaux_ocn2med_file1_doavg = .false.
+      histaux_ocn2med_file1_enabled = .true.
```

Comment:
As is, any test using this s2s will be trying to write histaux files every MED_history_n. While this still works as MED_history_n is a big number, I think we want a new atparse variable:
histaux_ice2med_file1_enabled = @[histaux_enabled]
histaux_ocn2med_file1_enabled = @[histaux_enabled]
with histaux_enabled=.false. in default_vars

Then turn it on only in tests/tests/cpld_control_nowave_noaero_p8 with `export histaux_enabled=.true.`

Reply:
See 441a09f. Does it look correct?